### PR TITLE
Marshall, Carter, and Dark Corporate Liaison (Actual GOI Rep)

### PR DESCRIPTION
**Pull request by AndJusticeForAll0**
MCD

## About the Pull Request

Adds MC&D Liaison and new fax location. This is an alternate title for GOC rep. 

## Why It's Good For The Game

Adding an  MC&D liaison is a great way for more RP to be enabled for the player. The MC&D are completely different compared to other GOIs, and provide a completely new gameplay loop. The MC&D rep would be able to attempt to bribe(they get a briefcase filled with cash), blackmail, or even persuade Foundation members to make deals. And while some people may say  MC&D is an antagonistic faction, this is not true. MCD often works with the Foundation and cuts deals with them, which would be a great thing to experience in game. 
## Changelog

:cl:
add: Adds new GoI Rep in the form of MCD
/:cl:

### CODE REVIEW

**Comment by checkraisefold** [comment]
mc&d has no purpose being in the facility. it's better than shark punching and the foundation has relations with mc&d, but its even worse than the uiu rep in having a lore reason for being there.

File: code/modules/asset_cache/assets/on_join.dm
Comment (checkraisefold) [inline code/modules/asset_cache/assets/on_join.dm]: why?? were you going to add mc&d logo to paper and just didn't? add the mc&d logo please

File: code/game/jobs/job/captain.dm
Comment (checkraisefold) [inline code/game/jobs/job/captain.dm:189]: who's the fucking joker that double defined the jobs in both the map job dm and the base game job dm files

File: maps/site53/job/outfits.dm
Comment (checkraisefold) [inline maps/site53/job/outfits.dm:493]: outfit's a bit lame

**Comment by Vsgramor** [comment]
i personally think this could be a cool alternative take on the GOI role, and i would even say its more impactful on creating alternative gameplay incentives for the role than UIU rep and horizon which are already in the server.

**Comment by ISOSS13** [comment]
Needs work. Good first PR but in future do more debugging before posting, even very simple stuff can have lots of issues. Lestat has stated we are not exactly following SCP lore, however I still feel it should be brought up that this fits into the SCP "lore"

File: code/game/jobs/job/captain.dm
Comment (ISOSS13) [reply code/game/jobs/job/captain.dm:189]: Bit of a tangent: Cant remember where this started originally but in total most jobs on Baycode are defined in 2-3 locations that do not point to eachother with comments or even fucking reference eachother directly. Manny probably knows (HoH dev if you don't know them). Not that I'd recommend asking over something trivial like that. Not having it defined in two places used to cause issues (probably does here too) and I'm saying all this without having glanced at SCP13s code more than once or twice.

File: code/modules/asset_cache/assets/on_join.dm
Comment (AndJusticeForAll0) [reply code/modules/asset_cache/assets/on_join.dm]: Fixed.

File: code/game/jobs/job/captain.dm
Comment (AndJusticeForAll0) [reply code/game/jobs/job/captain.dm:189]: While it may seem redundant, the entire thing breaks if you don't double define it.

**Comment by AndJusticeForAll0** [comment]
Everything has been fixed, review at your leisure.

File: code/game/jobs/job/captain.dm
Comment (checkraisefold) [reply code/game/jobs/job/captain.dm:189]: > While it may seem redundant, the entire thing breaks if you don't double define it.

yeah it's not your fault this is just a general question, wondering which joker did this

File: code/_helpers/text.dm
Comment (hry-gh) [inline code/_helpers/text.dm]:
```suggestion
```

remove conflict markers

**Comment by Vizoplez** [comment]
MC&D would never be allowed on a foundation site.

Maybe as an external fax /only/.

For reference, MCD actively go against the Foundation's objectives at pretty much every turn. Stealing, re-selling and creating anomalies, regularly creating breaches of normalcy, ecetera.

They're a rogue group, and I highly doubt they'd be every granted an actual office. Sure, lines of communication may be opened between the two, but never direct /visits/ without exception circumstance.

**Comment by AndJusticeForAll0** [comment]
"A lot of MC&D works involve cooperation with the Foundation, and more than a few have the two organizations at odds over certain items. Remember that, more often than not, the Foundation and MC&D are aligned in their tactics and core goals. MC&D might very well participate in saving the world alongside the Foundation. But when it's over, instead of putting the world back together they'll think how best to make a dollar off the ruins." 
Direct quote from GoI field guide. 
While they certainly aren't buddy-buddy, there's a good chance that at least one liaison exists between them. Hell, it would be less likely if ONE didn't exist. It's a pain to do all negotiations by faxes, which is why most countries have diplomats of countries they don't have positive relations with. I seriously don't think MC&D would harm gameplay/RP at all.

**Review by hry-gh — COMMENTED** [review]
outoutoutdated

**Comment by checkraisefold** [comment]
PR stale? conflicts + still waiting for thematic approval

**Review by projectkepler-ru — COMMENTED** [review]
@AndJusticeForAll0  
Can you change this to be a Merchant or something similar to that? I can change up the map abit to accomodate a shop-like area for em. We are abit anxious about having a full on MC&D Agent on site

**Review by hry-gh — APPROVED** [review]
received... thematic... approval

**Pull request merged**
